styles.css

```diff
@@ -93,3 +129,144 @@ nav {
 /* frontpage */
 
 /* -------------------------------- */
+
+/* projectssection */
+.projectsContainer {
+  display: grid;
+  row-gap: 4%;
+}
+
+.project1 {
+  background-color: #fff;
+  margin: 0 3% 0 3%;
+  border-radius: 20px;
+  border: 1px solid #dfe1e6;
+  padding-bottom: 3%;
+}
+
+.project1 ul {
+  list-style-type: none;
+  display: flex;
+  padding-left: 0;
+  margin-left: 8%;
+  gap: 2%;
+}
+
+.b li {
+  background-color: #ebebff;
+  padding: 2%;
+  border-radius: 10px;
+  color: #6070ff;
+}
+
+.a li:nth-child(1) {
+  color: #344563;
+}
+
+.a li:nth-child(3) {
+  color: #7a869a;
+}
+
+.a li:nth-child(5) {
+  color: #7a869a;
+}
+
+.project1 button {
+  padding: 3%;
+  margin-left: 8%;
+  margin-bottom: 4%;
+  border-radius: 10px;
+  border: 1px solid #396df2;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+button:active {
+  background-color: #2230d2;
+}
```

Comment:
- [ ] Well done implementing the button styling 🥇 Good job. Also, please make sure to fix the color for `active` styles as well as you did with the `hover` 😀⚒ This can easily be fixed by adding `color: #fff;` for `button:active` as well 💪

### _Button style now 🔽_
![image](https://user-images.githubusercontent.com/106119105/227199037-4d5c6384-7bc1-4d74-ba7d-67ca957e9b3c.png)

### _Figma design guidelines🔽_
![image](https://user-images.githubusercontent.com/106119105/227199246-7c459242-e0cf-4895-b432-8af3d588f61e.png)